src/mahalanobis.jl

```diff
@@ -1,11 +1,65 @@
 # Mahalanobis distances
 
+"""
+    Mahalanobis(Q; skipchecks=false) <: Metric
+
+Create a Mahalanobis distance (i.e., a bilinear form) with covariance matrix `Q`.
+Upon construction, both symmetry/self-adjointness and positive semidefiniteness are checked,
+unless the keyword argument `skipchecks = true`.
+
+# Example:
+```julia
+julia> A = collect(reshape(1:9, 3, 3)); Q = A'A;
+
+julia> dist = Mahalanobis(Q)
+Mahalanobis{Matrix{Int64}}([14 32 50; 32 77 122; 50 122 194])
+
+julia> dist = Mahalanobis(A)
+ERROR: ArgumentError: bilinear form is not symmetric/Hermitian
```

Comment:
Would be good to update the example I assume 🙂

Reply:
🤦